authprox/apadmin/test.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+DBG_TEST=2
+DBG_SRV=0
+
+. "$(go env GOPATH)/src/github.com/dedis/cothority/libtest.sh"
+
+main(){
+	build $APPDIR/../../byzcoin/bcadmin
+	startTest
+	buildConode github.com/dedis/cothority/byzcoin github.com/dedis/cothority/authprox
+
+	# This must succeed before the others will work.
+	run BCSetup
```

Comment:
Usually it's nice to have every test being able to run independently. This makes it easier to run them if something goes wrong...

Reply:
I will let the next person that's hacking on this decide on that for themselves. At this point, yagni.